### ISSUE TITLE
请问token是如何获取的？

### ISSUE THREAD

**Issue by wardenlym** [leeezep/kurobbs_auto_checkin#2]
我根据这个方法：
https://wave.tealc.fun/#/token

设置了TOKEN，但是每次都必定报错：
```python
2025-03-12 04:29:18.494 | DEBUG    | __main__:make_request:58 - {
  "code": 220,
  "msg": "登录已过期，请重新登录",
  "success": null
}
None
2025-03-12 04:29:18.494 | ERROR    | __main__:main:[16](https://github.com/wardenlym/kurobbs_auto_checkin/actions/runs/13803712404/job/38610611128#step:5:17)2 - An unexpected error occurred: 'NoneType' object is not subscriptable
Error: Process co
```

请问下您是怎么获取token才能跑成功的呢？

**Comment by leeezep** [comment]
请注意![Image](https://github.com/user-attachments/assets/63352250-3a83-4243-b51a-d9d827db3cd6)
另外最好使用app端的token
